### ISSUE TITLE
Unexported mesh collider

### ISSUE THREAD

**Issue by mydiscogr** [fehaar/FFWD#58]
hi, I've imported a 3d object via fbx, then I generated collision map, in unity all works fine, but when I export scene to xna "Unesported Meshe collider" appears you know why?

**Comment by fehaar** [comment]
Yes. It is because of FFWDs 2D physics system. Per default Mesh 
colliders are not supported - so they are not exported. If you do make 
a mesh collider that is in 2D and you want it exported, you can use the 
included XNAMeshExporter script and just put it on the object you want 
to export and it will be exported and added in the XNA physics.

There is also an experimental XNAPolygonCollider script that you can 
use to define polygons that will be exported to XNA as colliders. The 
editor functionality of the script is a bit buggy, but the exporting 
should work fine. So you should be able to fix the editor behaviour and 
have a usable script that is easier than making real meshes.

## 

Thomas Gravgaard
Senior Software Developer
Press Play ApS
thomas@pressplay.dk
+45 26 74 26 74
